style.css

```diff
@@ -0,0 +1,126 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.menu-toggle {
+  display: none;
+}
+
+.container {
+  background-color: #3e3c3c;
+  display: flex;
+  height: 72px;
+}
+
+.container .name-info {
+  flex: 3;
+}
+
+.container ul {
+  display: flex;
+  justify-content: space-around;
+  flex: 1;
+}
+
+.list-container li {
+  list-style: none;
+  text-decoration: none;
+}
+
+@media screen and (max-width: 600px) {
+  main {
+    background-color: #1c1a19;
+    padding: 20px;
+    height: 100vh;
+    background-image: url(./image/bg-img.png);
+    background-repeat: no-repeat;
+    background-position: 100% 90%;
+  }
+
+  nav {
+    width: 100%;
+    padding: 0;
+  }
+
+  .container {
+    background-color: #3e3c3c;
+    display: flex;
+    height: 90px;
+    padding: 10px;
+    align-content: center;
+    align-items: center;
+  }
+
+  .container .name-info {
+    display: flex;
+    flex-wrap: wrap;
+    font-style: normal;
+    font-weight: 700;
+    text-decoration: none;
+    font-size: 18px;
+    line-height: 24px;
+    color: #fff;
+    flex: 10;
+  }
+
+  .menu-toggle {
+    display: block;
+    flex: 1;
+  }
+
+  .container ul {
+    display: none;
+  }
+
+  .bio-container {
+    display: flex;
+    padding-top: 15vh;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+  }
+
+  .bio-container h1 {
+    display: flex;
+    color: #ff6b00;
+    flex-wrap: wrap;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 64px;
+    letter-spacing: 0.37px;
+    order: 0;
+    flex-grow: 0;
+    margin: 5px 0;
+  }
+
+  .paragragh p {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+    margin: 0;
+  }
+
+  .bio-container h3 {
+    color: #fff5e1;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    flex: 1;
+    align-items: center;
+    margin: 5px 0;
+  }
+
+  .icon {
+    display: flex;
+  }
+
+  .icon img {
+    padding: 10px;
+  }
+}
```

Comment:
[REQUIED]  Please set list-style to none in the list that your created for the icons
![image](https://user-images.githubusercontent.com/74744787/158780334-a3268b41-967d-4d61-a736-70f2a0202231.png)

Reply:
done